### PR TITLE
Add SEPTA rail contingency plan warning.

### PR DESCRIPTION
**Pull request by flibbertigibbet**
Display alert for SEPTA rail trips with link to contingency plan.

### CODE REVIEW

**Comment by flibbertigibbet** [comment]
image

File: src/app/scripts/cac/control/cac-control-directions-list.js [hunk 1]
```diff
@@ -93,6 +93,18 @@ CAC.Control.DirectionsList = (function (_, $, MapTemplates, Social, UserPreferen
...
+        // when modified rail schedule published.
+
+        // Show alert regarding current SEPTA rail status
+        _.every(itinerary.legs, function(leg) {
+            if (leg.transitLeg && leg.mode === 'RAIL' && leg.agencyName === 'SEPTA') {
+                $container.append(MapTemplates.septaRailWarningAlert());
+                return false; // break on first SEPTA rail leg found
+            }
+            return true;
+        });
```
Comment (KlaasH) [inline src/app/scripts/cac/control/cac-control-directions-list.js]: In the pantheon of very similar lodash functions, `some` might be more straightforward here, e.g.:
```
if(_.some(itinerary.legs, function(leg) {
    return leg.transitLeg && leg.mode === 'RAIL' && leg.agencyName === 'SEPTA'; })) {
    $container.append(MapTemplates.septaRailWarningAlert());
}
```
Thought the spacing on that comes out a little crunchy.

Either way works fine.

**Comment by KlaasH** [comment]
:+1:

File: src/app/scripts/cac/control/cac-control-directions-list.js (hunk 1 above)
Comment (flibbertigibbet) [inline src/app/scripts/cac/control/cac-control-directions-list.js]: Yup, that's a little cleaner.

**Pull request merged**